azext_iot/tests/test_iot_ext_unit.py

```diff
@@ -469,6 +496,142 @@ def test_device_update_error(self, serviceclient_generic_error, req):
                 parameters=req,
             )
 
+    @pytest.mark.parametrize(
+        "req",
+        [
+            generate_device_custom(generate_device_show(), False, "disabled"),
+            generate_device_custom(generate_device_show(), True, "enabled", "status_updated"),
+            generate_device_custom(
+                generate_device_show(
+                    authentication={
+                        "symmetricKey": {"primaryKey": "", "secondaryKey": ""},
+                        "type": "sas",
+                    }
+                ), True, "enabled", None, "shared_private_key", None, None, "primary_Updated", "secondary_Updated"
```

Comment:
we can pass only those arguments which are required

Reply:
updated as per suggested